scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,58 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT current_database() || ''_admin''::text;'
```

Comment:
Sorry, I did not notice that LANGUAGE is plain SQL, so I have nothing against it as an exception.

Probably the alternative is:
- do it in plpgsql (unless there's a major impediment)
- write sth like:
```sql
CREATE OR REPLACE
FUNCTION cartodb.CDB_Organization_Member_Group_Role_Member_Name()
    RETURNS TEXT
AS $$ 
  SELECT current_database() || '_admin';
$$
LANGUAGE SQL;
```
(you were adding extra quotes because of the delimiter chosen, more [here](http://www.postgresql.org/docs/9.4/static/plpgsql-development-tips.html#PLPGSQL-QUOTE-TIPS))
cleaner, isn't it?